pkg/volume/util/fs/fs_windows.go

```diff
@@ -75,3 +77,36 @@ func DiskUsage(path string) (*resource.Quantity, error) {
 func Find(path string) (int64, error) {
 	return 0, nil
 }
+
+func diskUsage(currPath string, info os.FileInfo) (int64, error) {
+	var size int64
+
+	dir, err := os.Open(currPath)
+	if err != nil {
+		return size, err
+	}
+	dirInfo, err := os.Lstat(currPath)
+	if err != nil {
+		return size, err
+	}
+	size += dirInfo.Size()
+	defer dir.Close()
```

Comment:
shouldn't this go up before 88 for the rare case Open() could succeed but LStat fails?